src/ai/backend/manager/models/vfolder.py

```diff
@@ -947,6 +951,11 @@ async def prepare_vfolder_mounts(
             extra_vf_conds,
             VFolderRow.id.in_(requested_mount_map.keys()),
         )
+    if requested_vfolder_ids:
+        extra_vf_conds = sa.or_(
+            extra_vf_conds,
+            VFolderRow.id.in_(requested_vfolder_ids),
+        )
```

Comment:
```suggestion
    requested_vfids = set(requested_mount_map.keys() or ()) | set(requested_vfolder_ids or ())
    if requested_vfids:
        extra_vf_conds = sa.or_(
            extra_vf_conds,
            VFolderRow.id.in_(requested_vfids),
        )
```

Reply:
Can you please reflect this code? @fregataa

Reply:
I'll leave it at that, but please reflect in your next task. @fregataa